ts/components/messages/MessageDueDateBar.tsx

```diff
@@ -33,6 +39,13 @@ type OwnProps = {
 
 type Props = OwnProps & ReturnType<typeof mapDispatchToProps>;
 
+type PaymentStatus =
```

Comment:
IMHO we used a misleading naming:

`expiredNotExpirable` it's very to hard to understand

Do you also find this type more descriptive?

```typescript
type PaymentStatus =
  | "paid"
  | "expiring"
  | "expiredNotValid"
  | "expiredButValid"
  | "valid";
```

Reply:
I agree it's hard to get, and the variant you proposed is more linear. The only concern I have is `valid` has a specific meaning which is not strictly related to `expirable`/`unexpirable`. Such concepts are already mapped [here](https://github.com/pagopa/io-app/blob/ab16e0dfc505c36bac603ed9f0ee95940b2249f4/ts%2Futils%2Fmessages.ts#L161).